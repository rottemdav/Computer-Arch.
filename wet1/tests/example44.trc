4 2 2 1 global_history global_tables using_share_mid
0x01fc T 0xb64170e0
0x8ee79c4 T 0x08cb90
0x3e1b4 N 0x76fc3e4
0xf3f3ece4 N 0x0874
0x1cc0 N 0x81f4afc
0xc5410c T 0xd5e0
0x04f0 T 0xc7914
0x228bca98 T 0x87b13ec0
0xc5410c N 0xd5e0
0x8ee79c4 N 0x08cb90
0x1cc0 N 0x81f4afc
0xfe98 N 0x2df724
0x41d8a914 N 0xe91e6690
0x1cc0 T 0x81f4afc
0xa987a0 N 0xf10ca88
0xc5410c N 0xd5e0
0x3e1b4 N 0x76fc3e4
0x8ee79c4 T 0x08cb90
0xa987a0 N 0xf10ca88
0x3e1b4 N 0x76fc3e4
0xc5410c N 0xd5e0
0xf3f3ece4 N 0x0874
0x2129a524 N 0x23a0
0xfe98 N 0x2df724
0x41d8a914 T 0xe91e6690
0x2129a524 T 0x23a0
0xf3f3ece4 T 0x0874
0x2129a524 N 0x23a0
0x04f0 N 0xc7914
0x41d8a914 N 0xe91e6690
0xfe98 N 0x2df724
0x228bca98 T 0x87b13ec0
0x01fc N 0xb64170e0
0x1cc0 N 0x81f4afc
0x16df4ef4 N 0x6bc
0x04f0 N 0xc7914
0x2129a524 T 0x23a0
0xf3f3ece4 N 0x0874
0x1cc0 N 0x81f4afc
0x1cc0 T 0x81f4afc
0xc5410c T 0xd5e0
0x8ee79c4 T 0x08cb90
0x01fc N 0xb64170e0
0x1a333c N 0xb6920284
0x16df4ef4 T 0x6bc
0x2129a524 N 0x23a0
0x1a333c N 0xb6920284
0xc5410c T 0xd5e0
0x41d8a914 T 0xe91e6690
0x2129a524 N 0x23a0
0x41d8a914 T 0xe91e6690
0x04f0 N 0xc7914
0x8ee79c4 T 0x08cb90
0x3e1b4 N 0x76fc3e4
0x3e1b4 T 0x76fc3e4
0x54d078a0 T 0x164
0x41d8a914 N 0xe91e6690
0x3e1b4 N 0x76fc3e4
0xc5410c N 0xd5e0
0x01fc T 0xb64170e0
0xa987a0 N 0xf10ca88
0x1cc0 N 0x81f4afc
0x1cc0 N 0x81f4afc
0xa987a0 N 0xf10ca88
0x16df4ef4 T 0x6bc
0x3294938 N 0xb90
0x8ee79c4 T 0x08cb90
0xa987a0 T 0xf10ca88
0x228bca98 T 0x87b13ec0
0x01fc N 0xb64170e0
0x3e1b4 N 0x76fc3e4
0xfe98 N 0x2df724
0x1a333c N 0xb6920284
0x54d078a0 T 0x164
0xc5410c N 0xd5e0
0x04f0 T 0xc7914
0x1a333c T 0xb6920284
0xfe98 T 0x2df724
0x41d8a914 T 0xe91e6690
0xfe98 T 0x2df724
0xf3f3ece4 N 0x0874
0x16df4ef4 T 0x6bc
0x01fc N 0xb64170e0
0x16df4ef4 N 0x6bc
0xf3f3ece4 T 0x0874
0xc5410c T 0xd5e0
0xa987a0 N 0xf10ca88
0x54d078a0 T 0x164
0x01fc N 0xb64170e0
0x04f0 T 0xc7914
0x8ee79c4 T 0x08cb90
0xfe98 N 0x2df724
0x04f0 T 0xc7914
0x01fc N 0xb64170e0
0x1a333c N 0xb6920284
0xc5410c N 0xd5e0
0xf3f3ece4 T 0x0874
0x3294938 T 0xb90
0xfe98 N 0x2df724
0x228bca98 T 0x87b13ec0
0x1a333c N 0xb6920284
0x3e1b4 N 0x76fc3e4
0x41d8a914 N 0xe91e6690
0x41d8a914 T 0xe91e6690
0x228bca98 T 0x87b13ec0