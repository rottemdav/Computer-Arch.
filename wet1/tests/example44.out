0x1fc N 0x200
0x8ee79c4 N 0x8ee79c8
0x3e1b4 N 0x3e1b8
0xf3f3ece4 N 0xf3f3ece8
0x1cc0 N 0x1cc4
0xc5410c N 0xc54110
0x4f0 N 0x4f4
0x228bca98 N 0x228bca9c
0xc5410c N 0xc54110
0x8ee79c4 N 0x8ee79c8
0x1cc0 N 0x1cc4
0xfe98 N 0xfe9c
0x41d8a914 N 0x41d8a918
0x1cc0 N 0x1cc4
0xa987a0 N 0xa987a4
0xc5410c T 0xd5e0
0x3e1b4 N 0x3e1b8
0x8ee79c4 N 0x8ee79c8
0xa987a0 N 0xa987a4
0x3e1b4 N 0x3e1b8
0xc5410c N 0xc54110
0xf3f3ece4 N 0xf3f3ece8
0x2129a524 N 0x2129a528
0xfe98 N 0xfe9c
0x41d8a914 N 0x41d8a918
0x2129a524 N 0x2129a528
0xf3f3ece4 T 0x23a0
0x2129a524 N 0x2129a528
0x4f0 N 0x4f4
0x41d8a914 N 0x41d8a918
0xfe98 T 0x2df724
0x228bca98 N 0x228bca9c
0x1fc N 0x200
0x1cc0 N 0x1cc4
0x16df4ef4 N 0x16df4ef8
0x4f0 N 0x4f4
0x2129a524 N 0x2129a528
0xf3f3ece4 N 0xf3f3ece8
0x1cc0 N 0x1cc4
0x1cc0 N 0x1cc4
0xc5410c N 0xc54110
0x8ee79c4 N 0x8ee79c8
0x1fc N 0x200
0x1a333c T 0xb64170e0
0x16df4ef4 N 0x16df4ef8
0x2129a524 N 0x2129a528
0x1a333c N 0x1a3340
0xc5410c N 0xc54110
0x41d8a914 N 0x41d8a918
0x2129a524 N 0x2129a528
0x41d8a914 N 0x41d8a918
0x4f0 N 0x4f4
0x8ee79c4 N 0x8ee79c8
0x3e1b4 N 0x3e1b8
0x3e1b4 T 0x76fc3e4
0x54d078a0 N 0x54d078a4
0x41d8a914 N 0x41d8a918
0x3e1b4 N 0x3e1b8
0xc5410c T 0xd5e0
0x1fc N 0x200
0xa987a0 T 0x164
0x1cc0 N 0x1cc4
0x1cc0 N 0x1cc4
0xa987a0 N 0xa987a4
0x16df4ef4 N 0x16df4ef8
0x3294938 N 0x329493c
0x8ee79c4 N 0x8ee79c8
0xa987a0 N 0xa987a4
0x228bca98 N 0x228bca9c
0x1fc N 0x200
0x3e1b4 N 0x3e1b8
0xfe98 T 0x87b13ec0
0x1a333c N 0x1a3340
0x54d078a0 T 0xf10ca88
0xc5410c N 0xc54110
0x4f0 N 0x4f4
0x1a333c N 0x1a3340
0xfe98 N 0xfe9c
0x41d8a914 N 0x41d8a918
0xfe98 T 0x2df724
0xf3f3ece4 N 0xf3f3ece8
0x16df4ef4 N 0x16df4ef8
0x1fc N 0x200
0x16df4ef4 N 0x16df4ef8
0xf3f3ece4 N 0xf3f3ece8
0xc5410c N 0xc54110
0xa987a0 N 0xa987a4
0x54d078a0 N 0x54d078a4
0x1fc N 0x200
0x4f0 N 0x4f4
0x8ee79c4 N 0x8ee79c8
0xfe98 T 0x2df724
0x4f0 T 0xc7914
0x1fc N 0x200
0x1a333c T 0xb64170e0
0xc5410c N 0xc54110
0xf3f3ece4 N 0xf3f3ece8
0x3294938 N 0x329493c
0xfe98 N 0xfe9c
0x228bca98 N 0x228bca9c
0x1a333c N 0x1a3340
0x3e1b4 N 0x3e1b8
0x41d8a914 N 0x41d8a918
0x41d8a914 N 0x41d8a918
0x228bca98 T 0x87b13ec0
flush_num: 49, br_num: 105, size: 142b
